### PR TITLE
Update StoryBookParagraph.java

### PR DESCRIPTION
**Pull request by watr-sheep**
Update on the issue - #1357 refactor: Remove audio from StoryBookParagraph.

### CODE REVIEW

**Review by nya-elimuai — CHANGES_REQUESTED** [review]
@watr-sheep Thank you for your contribution 🙂

Note that you also need to update the corresponding JSP files: https://github.com/elimu-ai/webapp/tree/master/src/main/webapp/WEB-INF/jsp/content/storybook/paragraph

**Comment by watr-sheep** [comment]
> @watr-sheep Thank you for your contribution 🙂
> 
> Note that you also need to update the corresponding JSP files: https://github.com/elimu-ai/webapp/tree/master/src/main/webapp/WEB-INF/jsp/content/storybook/paragraph

All done. Pls check if I missed anything.

**Review by nya-elimuai — CHANGES_REQUESTED** [review]
@watr-sheep Yes, these changes look good, thank you 👍

The only missing piece now is adding a database migration script for deleting the column from the underlying database table. See https://github.com/elimu-ai/webapp/tree/master/src/main/resources/db/migration for previous examples.

**Comment by watr-sheep** [comment]
Hi @nya-elimuai, It's been a while. I've done the required changes related to db migration script.

File: src/main/resources/db/migration/2001224.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# 2.1.224
+
+ALTER TABLE StoryBookParagraph DROP COLUMN audio;
```
Comment (nya-elimuai) [inline src/main/resources/db/migration/2001224.sql:3]: @watr-sheep The column name needs to be adjusted:

```
> DESC StoryBookParagraph;
+---------------------+---------------+------+-----+---------+----------------+
| Field               | Type          | Null | Key | Default | Extra          |
+---------------------+---------------+------+-----+---------+----------------+
| id                  | bigint(20)    | NO   | PRI | NULL    | auto_increment |
| originalText        | varchar(1024) | NO   |     | NULL    |                |
| sortOrder           | int(11)       | NO   |     | NULL    |                |
| storyBookChapter_id | bigint(20)    | YES  | MUL | NULL    |                |
| audio_id            | bigint(20)    | YES  | MUL | NULL    |                |
+---------------------+---------------+------+-----+---------+----------------+
```

**Review by nya-elimuai — CHANGES_REQUESTED** [review]

File: src/main/resources/db/migration/2001224.sql (hunk 1 above)
Comment (watr-sheep) [reply src/main/resources/db/migration/2001224.sql]: Fixed it.

**Review by nya-elimuai — APPROVED** [review]
@watr-sheep Looks good 👍

Ready to be merged.

**Comment by watr-sheep** [comment]
@nya-elimuai I was wondering what happens to the deleted audio attribute's references left in the controllers. Should those be removed also?

**Pull request merged**